src/ImageDimensionsProcessor.php

```diff
@@ -0,0 +1,25 @@
+<?php
+
+namespace Phauthentic\Infrastructure\Storage\Processor\Image;
+
+use Phauthentic\Infrastructure\Storage\FileInterface;
+use Phauthentic\Infrastructure\Storage\Processor\ProcessorInterface;
+
+class ImageDimensionsProcessor implements ProcessorInterface
```

Comment:
Should we name this then LocalImageDimensionsProcessor, if this can only work on local files?
Or does it not matter here?